test_runner/regress/test_fullbackup.py

```diff
@@ -61,7 +51,7 @@ def test_fullbackup(
     # use resetwal to overwrite it
     pg_resetwal_path = os.path.join(pg_bin.pg_bin_path, "pg_resetwal")
     cmd = [pg_resetwal_path, "-D", str(restored_dir_path)]
-    pg_bin.run_capture(cmd, env=psql_env)
```

Comment:
pg_bin already sets the LD_LIBRARY_PATH, but I guess no one noticed when introducing it.